addons/survey/models/survey_survey.py

```diff
@@ -382,6 +383,7 @@ def copy(self, default=None):
         }
         for src, dst in zip(self.question_ids, clone.question_ids):
             if src.is_conditional:
+                dst.is_conditional = True
```

Comment:
Well you lost me. If we already had a way to copy properly the conditional configuration, why not keep the copy=True ?I guess there is a reason why you did that but I don't get it. Can you explain to my poor brain ?